src/screens/PodcastScreen.tsx

```diff
@@ -19,12 +26,29 @@ export class PodcastScreen extends React.Component<Props, State> {
 
   constructor(props: Props) {
     super(props)
+    const podcast = this.props.navigation.getParam('podcast')
+
     this.state = {
-      fromSelected: 'Downloaded',
+      clips: [],
+      episodes: [],
+      fromSelected: 'All Episodes',
+      isLoading: true,
+      podcast: podcast ? podcast : {},
```

Comment:
This looks like it won't change while the PodcastScreen is up so maybe you don't need it to be in state. You can access it through props with a default value if its not there `this.props.navigation.getParam('podcast', {})`

Reply:
Ok removed it from state.